.github/workflows/docker-server.yml

```diff
@@ -45,7 +46,7 @@ jobs:
 
       - name: Push Commit image to DockerHub
         uses: docker/build-push-action@v2
-        if: github.event_name == 'push'
+        if: ${{ !env.ACT }} && github.event_name == 'push'
```

Comment:
As far as I can tell, this should be `if: !env.ACT && github.event_name == 'push'`?

Reply:
this gives line 49: did not find expected alphabetic or numeric character

Reply:
try `github.event_name == 'push' && !env.ACT` ? I think the `!` may be confusing the yaml parser

Reply:
yea seems this was the case